windows/sound.cpp

```diff
@@ -560,13 +560,10 @@ CSound::CSound ( void           (*fpNewCallback) ( CVector<int16_t>& psData, voi
     // in case we do not have a driver available, throw error
     if ( lNumDevs == 0 )
     {
-        throw CGenErr ( "<b>" + tr ( "No ASIO audio device (driver) found." ) + "</b><br><br>" +
-            tr ( "The " ) + APP_NAME + tr ( " software requires the low latency audio "
-            "interface ASIO to work properly. This is not a standard "
-            "Windows audio interface and therefore a special audio driver is "
-            "required. Either your sound card has a native ASIO driver (which "
-            "is recommended) or you might want to use alternative drivers like "
-            "the ASIO4All driver." ) );
+        throw CGenErr ( "<b>" + tr ( "No audio device with ASIO driver detected" ) + "</b><br><br>" +

```

Comment:
I prefer the original `No ASIO audio device (driver) found.`, to be honest, for the first line.  (It's got a full stop, too :) .)

Reply:
Yes, this is shorter but maybe not that clear for a new user who doesn’t know anything about ASIO. 

Tagging @mulyaj here who proposed this exact wording.

Reply:
Whether they know about ASIO doesn't make one preferable to the other, as both describe the problem as being with something to do with ASIO.  The point is, it's not the _device_, it's the _driver.  Your phrasing changes the emphasis.  Indeed, the original (with the `()`) had it wrong, in my view.  But it had a full stop...

Reply:
My initial thought was "Install an ASIO driver to use Jamulus" which empathises the solution rather then the error. I think muljay didn’t like it because it doesn’t clearly explain the error.

Reply:
> Your phrasing changes the emphasis

That's right, I agree with @pljones here.